products/accessories.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: default
+title: Subnero Modem Accessories
+banner : images/banner-accessories.jpg
+thumbnail: images/thumbnail-accessories.jpg
+categories: accessories
+section_id: products
+excerpt: Optional upgrades and accessories for Subnero underwater modems.
+---
+
+<div class='full tall' style='background-image: url({{site.baseurl}}/{{page.banner}});'>
+  <div class='row'>
+    <div class='large-12 columns'>
+      <!-- {% include section-header.html title=page.title tagline=page.tagline color=page.title_color class="big" %} -->
+    </div>
+  </div>
+  <div class='four spacing'></div>
+  <div class='four spacing'></div>
+</div>
+
+<div class='cGBxoB'>
+<section class='bsPRnx'>
+	<h1 class='thin' style='text-align: center'>Optional Accessories and Upgrades</h1>
+	<div class='eyXpDN'>
+		<div class='cmXrEt'>
+			<h1 class='thin' style='text-align: center'>Battery Packs</h1>
+			<ul class="gfXsQG">
+				<li class="fuqHMA">
```

Comment:
Same with class `fuqHMA`?

Reply:
Not sure I understand :(

Reply:
It's best practice to give CSS classes names that make sense. For example `class="col-xs-6` which implies this is a `column` of `xs` size and 6 points in width. But `fuqHMA` doesn't mean anything.

Reply:
These classes were taken from the reference provided during the revamping of the product listing page and were used as it is. I have raised an issue to rename all the classes (https://github.com/subnero1/subnero1.github.io/issues/148).